FhirToDataLake/SynapseValidate/Synapse_Validator.py

```diff
@@ -0,0 +1,207 @@
+import os
```

Comment:
nit, maybe we don't need the `_` in filename, we can name it `synapsevalidator.py`

Reply:
Sure, renamed to ```synapsevalidator.py```